tests/foreman/cli/test_activationkey.py

```diff
@@ -1726,26 +1587,28 @@ def test_positive_content_override(self, override_value):
 
         """
         if not bz_bug_is_open(1180282):
-            self.fail('BZ 1180282 has been closed. This test should be '
-                      'updated to verify value change via product-content.')
-
+            self.fail(
+                'BZ 1180282 has been closed. This test should be updated to'
+                'verify value change via product-content.'
+            )
         if self.pub_key is None:
             self._make_public_key()
-
-        result = ActivationKey.product_content({
-            u'id': self.pub_key['key_id'],
-            u'organization-id': self.pub_key['org_id'],
-        })
-        self.assertEqual(result.return_code, 0)
-        self.assertEqual(len(result.stderr), 0)
-        product_label = result.stdout[3].split('|')[5].replace(' ', '')
-
-        result = ActivationKey.content_override({
-            u'id': self.pub_key['key_id'],
-            u'organization-id': self.pub_key['org_id'],
-            u'content-label': product_label,
-            u'value': override_value,
-        })
-        self.assertEqual(result.return_code, 0)
-        self.assertEqual(len(result.stderr), 0)
-        self.assertIn(result.stdout[0], 'Updated content override')
+        for override_value in (u'1', u'0'):
+            with self.subTest(override_value):
+                result = ActivationKey.product_content({
+                    u'id': self.pub_key['key_id'],
+                    u'organization-id': self.pub_key['org_id'],
+                })
+                self.assertEqual(result.return_code, 0)
+                self.assertEqual(len(result.stderr), 0)
+                product_label = result.stdout[3].split('|')[5].replace(' ', '')
```

Comment:
would it be possible to separate the logic of the test up to this point from the subtest? That way we can reduce the number of times we perform this section. I think the most important part is the content override below (as well as the assertions)